mkdocs/docs/HPC/getting_started.md

```diff
@@ -0,0 +1,209 @@
+
+```shell
+$ ls
+run.sh TensorFlow-2.x_mnist-test.py
+```
+
+!!! Warning
+
+    When you do not see these files, make sure you uploaded the files to your **Home Directory**.
+
+### Submitting a job
+
+Jobs are submitted and executed using job scripts. In our case **run.sh**.
+
+A job script is a shell script, a text file that specifies the resources, software that is used (via module load) and the steps that should be done for the calculation.
+
+Our job script looks like this:
+
+<center>-- run.sh --</center>
+
+```bash
+#!/bin/bash
+
+module load TensorFlow/2.3.1-foss-2020a-Python-3.8.2
+
+time python TensorFlow-2.x_mnist-test.py
+
+```
+<sub>As you can see this job script will run the python program: **TensorFlow-2.x_mnist-test.py**</sub>
+
+
+The jobs you submit are per default executed on **cluser/{{defaultcluster}}**, you can swap cluster by the following command.
+
+<pre><code>module swap cluster/{{othercluster}}</code></pre>
+
+
+This job script can now be submitted to the cluster's job system for execution, using the qsub (**q**ueue **sub**mit) command:
+
+```shell
+$ qsub run.sh
+{{jobid}}
+```
+
+This command returns a job identifier (*{{jobid}}*) on the HPC cluster. this is a unique identifier for the job and can be used to monitor and manage your job.
+
+!!! Warning
+ 
+    Please take note that the module commands exclusively modify environment variables. For instance, using `module swap cluster/{{othercluster}}` will instruct `qsub` to carry out the task within the {{othercluster}} cluster. 
+    It's important to understand that while these commands affect jobs, they do not alter the context of commands executed directly in your shell. These commands will still operate on your login cluster: {{defaultcluster}}.
+
+For detailed information about `module` commands, read the [running batch jobs](running_batch_jobs.md) chapter.
+
+### Wait for job to be executed
+
+Your job is first put into a queue before being executed, so it can take a while.
+(see [when will my job start?](running_batch_jobs.md#when-will-my-job-start) for scheduling policy)
+
+You can get an overview of the active jobs using the `qstat` command:
+<pre><code>$ qstat
+Job ID     Name             User            Time Use S Queue
+---------- ---------------- --------------- -------- - -------
+{{jobid}}     run.sh           {{userid}}        0:00:00  Q doduo
+</code></pre> 
+
+Read [this section](running_batch_jobs.md#monitoring-and-managing-your-jobs) on how to interpret the output.
+
+### Inspect your results
+
+When your job finishes it generates 2 output files: For *stdout* and *stderr*.
+By default located in the directory where you issued qsub.
+
+In our example when running <code>ls</code> in the current directory you should see 2 new files:
+ 
+- **run.sh.e{{jobid}}**: stderr
+- **run.sh.o{{jobid}}**: stdout
+
+When examining the contents of ``run.sh.o{{jobid}}`` you will see something like this:
```

Comment:
Also mention that `run.sh.e*` should be mostly empty (no error messages).

If you remove the `time` from `run.sh` it may even be totally empty?